distributed/comm/tests/test_ucx.py

```diff
@@ -296,7 +296,7 @@ async def test_ucx_localcluster(processes, cleanup):
 
 
 @pytest.mark.slow
-@pytest.mark.asyncio
+@gen_test(timeout=60)
```

Comment:
Why is the timeout necessary? I wonder if this test gets into blocked states which necessitate the timeout?

Reply:
gen_test defaults to a 30 second timeout, this generally takes 24 seconds but I've seen it take 32